account_asset_management/models/account_move.py

```diff
@@ -60,8 +60,10 @@ def unlink(self):
 
     def write(self, vals):
         if set(vals).intersection(FIELDS_AFFECTS_ASSET_MOVE):
-            deprs = self.env["account.asset.line"].search(
-                [("move_id", "in", self.ids), ("type", "=", "depreciate")]
+            deprs = (
+                self.env["account.asset.line"]
+                .sudo()
```

Comment:
maybe better to use with_user(SUPERUSER_ID)

Reply:
No, the correct form is `sudo`. The other is when it's another different from SUPERUSER_ID.

Reply:
with_user : Return a new version of this recordset attached to the given user
this is for a write operation hence we want to know in the chatter who performed it.

Reply:
But this is not what we want to do. It's just bypass the security restriction, but not to put `OdooBot` on the chatter... Anyways, there's still the doubt why there's the access problem.

Reply:
@pedrobaeza 
As usual you are right, hence ignore my comment.
Hence now we'll wait on feedback from @Saran440 on how to reproduce the access problem.